.scripts/common.ts

```diff
@@ -107,10 +107,6 @@ function isPackageFolderPath(folderPath: string, packagesToIgnore: string[]): bo
   const packageJsonFilePath: string = joinPath(folderPath, "package.json");
   if (fileExistsSync(packageJsonFilePath)) {
     const packageJson: PackageJson = readPackageJsonFileSync(packageJsonFilePath);
-    // Skip all perf framework projects from gulp pack
-    if (packageJson.name.startsWith("@azure-tests/")) {
-      return false;
-    }
```

Comment:
Can you do something like below instead?
```
    // Skip all perf framework projects from gulp pack
    if (packageJson?.name?.startsWith("@azure-tests/")) {
      return false;
    }
```

Reply:
sure. changed.